e2e/discussions.spec.ts

```diff
@@ -282,10 +320,18 @@ test.describe('Discussions', () => {
           })
 
           await testAuth.step('Post a reply', async () => {
-            await validatedUserPage
-              .getByRole('textbox', { name: 'Write a reply...' })
-              .click()
-            await getThreadTextbox(validatedUserPage).fill(threadReply)
+            const replyInput = discussionThread.getByRole('textbox', {
+              name: 'Write a reply...',
+            })
+
+            // A second reply input briefly appears when the network
+            // connection is slow. Wait for the second input to be hidden.
+            await expect(replyInput).toHaveCount(1)
```

Comment:
Handle case where a second reply input briefly appears, as shown in Chrome when using "Slow 3G" network throttling --

https://github.com/Sage-Bionetworks/SynapseWebClient/assets/26949006/4063b7ae-c15a-43e0-a75d-0ba311cb11e3

Reply:
Super weird case! Nice catch!!